boards/nordic/nrf54l15dk/board.cmake

```diff
@@ -0,0 +1,11 @@
+# Copyright (c) 2024 Nordic Semiconductor ASA
+# SPDX-License-Identifier: Apache-2.0
+
+if (CONFIG_SOC_NRF54L15_CPUAPP)
+  board_runner_args(jlink "--device=cortex-m33" "--speed=4000")
+elseif (CONFIG_SOC_NRF54L15_CPUFLPR)
```

Comment:
```suggestion
if(CONFIG_SOC_NRF54L15_CPUAPP)
  board_runner_args(jlink "--device=cortex-m33" "--speed=4000")
elseif(CONFIG_SOC_NRF54L15_CPUFLPR)
```

Reply:
PDK configuration needs to be updated as well